packages/shared/forks/ReactFeatureFlags.native-fb.js

```diff
@@ -53,7 +53,7 @@ export const enableSuspenseLayoutEffectSemantics = false;
 export const enableNewReconciler = false;
 export const deferRenderPhaseUpdateToNextBatch = true;
 
-export const enableStrictEffects = false;
+export const enableStrictEffects = __DEV__;
 export const createRootStrictEffectsByDefault = false;
```

Comment:
Still need to decide what the right behavior is here, but this can wait on the concurrent root API decision.